notebooks/sippy-analysis.ipynb

```diff
@@ -0,0 +1,2411 @@
+{
```

Comment:
<p>I would expect the following work would be plots or graphs for each of these items above (Job Pass Rates By Variant, Top Failing Tests Without a Bug,etc). I think that is still missing here.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>